libraries/ui-library/src/global/base/colors.css

```diff
@@ -35,5 +35,5 @@
 
   --six-color-inactive: #c3c3c3;
 
-  --six-focus-ring-color: var(--six-color-web-rock-400);
+  --six-focus-ring-color: var(--six-color-action-600);
```

Comment:
I think the color code for hover that should be used or added is this one: #c52c0f

Reply:
I'm not entirely certain about your specific concern, but this color is intended to indicate focus for keyboard navigation, not hover. It's consistent with the color we use for focus on form elements. Or do you recommend to use `#c52c0f` for focus in general?
![image](https://github.com/six-group/six-webcomponents/assets/1475511/2877c917-9efb-429b-9212-382e902de2a3)
![image](https://github.com/six-group/six-webcomponents/assets/1475511/4c5d8a5b-2f8d-4028-97ca-10064eb28a8d)